student.rb

```diff
@@ -5,6 +5,11 @@ def initialize(age, classroom, name: 'Unknown', parent_permission: true)
     @classroom = classroom
```

Comment:
- [ ] Getters and Setters for the instance variable `@classroom` should be present.